README.md

```diff
@@ -16,10 +16,13 @@ Mumble is an Open Source, low-latency and high-quality voice-chat program
 written on top of Qt and Opus.
 
 There are two modules in Mumble; the client (mumble) and the server
-(murmur). The client works on Win32/64, Linux and Mac OS X, while the
+(murmur). The client works on Windows, Linux, FreeBSD and macOS, while the
```

Comment:
Why the "(murmur)" in front of this statement? It talks about client and server so I think the tag is a bit confusing